src/DeckGlMap/index.js

```diff
@@ -328,7 +335,13 @@ const DeckGlMap = ({
 					views={new MapView({repeat: true})}
 					viewState={deckView}
 					layers={[...finalBackgroundLayers, ...finalLayers]}
-					controller={true}
+					controller={
+						controller || controller === false
+							? controller
+							: {
+									dragRotate: false,
+							  }
+					}
```

Comment:
So there are 3 states? First, when controller is object, second, when controller is false, and third, when conroller is undefinded? What is the difference between false and undefined in this case? Please add comments to the code

Reply:
controller coul be bool or object, it is also in proptypes (i hope)

Reply:
Yes, but why? What is the difference between true, false and undefined in this case?